README.md

```diff
@@ -40,7 +40,7 @@ Terraform `terraform.tfvars` files contain sensitive information, so are encrypt
 
 ## Kubernetes resource management
 
-All [Kubernetes][kubernetes] resources are managed as [Helm][helm] charts, the Kubernetes package manager. Analytics-specific charts are served via our [Helm repository](https://ministryofjustice.github.io/analytics-platform-helm-charts/charts/) - source code is in the [ministryofjustice/analytics-platform-helm-charts](https://github.com/ministryofjustice/analytics-platform-helm-charts) repository, and chart values for each environment are stored in the [ministryofjustice/analytics-platform-config](https://github.com/ministryofjustice/analytics-platform-config) repository.
+All [Kubernetes][kubernetes] resources are managed as [Helm][helm] charts, the Kubernetes package manager. Analytics-specific charts are served via our [Helm repository](http://moj-analytics-helm-repo.s3-website-eu-west-1.amazonaws.com) - source code is in the [ministryofjustice/analytics-platform-helm-charts](https://github.com/ministryofjustice/analytics-platform-helm-charts) repository, and chart values for each environment are stored in the [ministryofjustice/analytics-platform-config](https://github.com/ministryofjustice/analytics-platform-config) repository.
 
```

Comment:
Sorry to be pedantic 
"and chart values for each environment..." might be "and chart secrets for each environment.." or "secrets and values"

Reply:
I'll leave that for a different PR - this one is about the Helm repo URL